src/main/java/rosenfeld/boggle/BoggleBoard.java

```diff
@@ -0,0 +1,56 @@
+package rosenfeld.boggle;
+
+import java.util.*;
+
+public class BoggleBoard {
+
+    private final int CUBE_SIDES = 6;
+    private int boardSize;
+    private List<String> cubeSet;
+    private Random random = new Random();
+    private Stack<String> stack = new Stack<>();
+
+    public BoggleBoard(int size) {
+        cubeSet = fourByFourCubes();
+        shuffleCubes();
+    }
+
+    public List<String> fourByFourCubes() {
+        return Arrays.asList(
+                "AAEGN", "ABBJOO", "ACHOPS", "AFFKPS",
```

Comment:
Where are these cubes from?

Reply:
I found a version here https://stanford.edu/class/archive/cs/cs106x/cs106x.1132/handouts/17-Assignment-3-Boggle.pdf

Reply:
If you are going to use online resources, ALWAYS include a link to them in your code.

There is a subtle bug here.